### ISSUE TITLE
Quotes are not escaped in checkstyle output

### ISSUE THREAD

**Issue by rapidrapids** [eslint/eslint#454]
Found in 0.1.4

Write some code with a missing `'use strict';`, in order to cause a warning/error. Run eslint with checkstyle format.

The output does not escape quotes (not sure if this should be considered a fault with the reporter or the rule):

``` xml
<error line="38" column="20" severity="warning" message="Missing "use strict" statement." />
```
